axolotl-web/src/components/Message.vue

```diff
@@ -233,26 +234,22 @@ export default {
       ); // #14 is the flag for quoting messages
 	    // see this list for all message types: https://github.com/nanu-c/axolotl/blob/main/app/helpers/models.go#L15
     },
-    isPlaying(){
-      return this.audio ? !this.audio.paused : false;
-    }
   },
   mounted() {
     if (this.message.Attachment!="" && this.message.Attachment!=null) {
       const attachment = JSON.parse(this.message.Attachment);
       if (attachment?.length>0 && attachment[0].CType === 3) {
-        this.audio = { 
-          ...new Audio("http://localhost:9080/attachments?file=" + attachment[0].File),
-          onloadedmetadata() {
+        this.audio = new Audio("http://localhost:9080/attachments?file=" + attachment[0].File);
+        var that = this;
+        this.audio.onloadedmetadata = function(){
           that.duration = that.audio.duration.toFixed(2);
-          },  
-          onended() {
-            that.audio.currentTime = 0;
-            that.isPlaying = false;
-          },
-          ontimeupdate() {
-            that.currentTime = that.audio.currentTime.toFixed(2);
-          }
```

Comment:
Maybe this didn't work because `that` was not defined.
Defining `var that = this;` before `this.audio = ...` could make it work.

Reply:
The error i tried to solve was `this.audio.pause` is not a function, meaning it's undefined.